packages/components/src/custom-select-control-v2/legacy-component/index.tsx

```diff
@@ -121,4 +121,15 @@ function CustomSelectControl( props: LegacyCustomSelectProps ) {
 	);
 }
 
-export default CustomSelectControl;
+export function ClassicCustomSelectControlV2Adapter(
+	props: LegacyCustomSelectProps
+) {
+	return (
+		<CustomSelectControl
+			__experimentalShowSelectedHint={ false }
```

Comment:
Not sure about the purpose of setting `__experimentalShowSelectedHint` to `false` in here, especially since it can be overridden by by `props` — can we just set the default value to `false` when deconstructing props in the component's implementation?

Reply:
Switched to setting default `false` value during prop destructuring